Dockerfile

```
@@ -4,35 +4,67 @@ FROM debian:bullseye-slim
 
 ARG VERSION
 
+###
 # For a list of pre-defined annotation keys and value types see:
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
+#
 # Note: Additional labels are added by the build workflow.
-LABEL org.opencontainers.image.authors="mark.feldhousen@cisa.dhs.gov"
+###
+LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
 LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
 
-RUN apt-get update && \
-DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
-ca-certificates \
-diceware \
-dovecot-imapd \
-dovecot-lmtpd \
-gettext-base \
-mailutils \
-opendkim \
-opendkim-tools \
-opendmarc \
-postfix \
-procmail \
-sasl2-bin \
-&& apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+###
+# This Docker container does not use an unprivileged user because it
+# must be able to modify postfix and opendkim config files and
+# therefore must run as root.
+###
 
+###
+# Upgrade the system
+###
+RUN apt-get update --quiet --quiet \
+    && apt-get upgrade --quiet --quiet
+
+###
+# Install everything we need
+###
+ENV DEPS \
+    ca-certificates \
+    diceware \
+    dovecot-imapd \
+    dovecot-lmtpd \
+    gettext-base \
+    mailutils \
+    opendkim \
+    opendkim-tools \
+    opendmarc \
+    postfix \
+    procmail \
+    sasl2-bin
+RUN DEBIAN_FRONTEND=noninteractive \
+    apt-get install --quiet --quiet --yes \
+    --no-install-recommends --no-install-suggests \
+    $DEPS
```

I see no reason why this should not continue to be one `RUN` command.

```suggestion
    $DEPS \
    && apt-get --quiet --quiet clean \
    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```